src/opnsense/mvc/app/controllers/OPNsense/Routing/forms/dialogEditGateway.xml

```diff
@@ -58,17 +58,17 @@
     </field>
     <field>
         <id>gateway_item.monitor_killstates</id>
-        <label>Kill states when down</label>
+        <label>Failover States</label>
         <type>checkbox</type>
         <style>monitor_opt</style>
-        <help>When a monitor down event is triggered, kill all states to this gateway.</help>
+        <help>Move clients to another available working gateway when this gateway is down. This requires default gatway switching or an assigned gateway group to trigger.</help>
```

Comment:
"another available working gateway" is a bit convoluted, but "up" isn't such a nice word and no suitable failover gateway may be available either

Reply:
"online" perhaps? At least that provides some symmetry with the gateway groups. If we define "working" as something more, we need to be even more explicit

Reply:
yeah "online" sounds good